File: Notes/app/src/main/java/com/serhii/apps/notes/ui/data_model/AuthModel.java
package com.serhii.apps.notes.ui.data_model;

import com.serhii.apps.notes.control.types.AuthorizeType;

public class AuthModel {

    private String email;
    private String password;
    private String confirmPassword;

    private AuthorizeType authType;

    public AuthModel() {
        this.email = "";
        this.password = "";
        this.confirmPassword = "";
    }

    public AuthModel(AuthorizeType authType) {
        this.authType = authType;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getConfirmPassword() {
        return confirmPassword;
    }

    public void setConfirmPassword(String confirmPassword) {
        this.confirmPassword = confirmPassword;
    }

    public void setAuthType(AuthorizeType authType) { this.authType = authType; }

    public AuthorizeType getAuthType() { return authType; }

}

File: Notes/app/src/main/java/com/serhii/apps/notes/ui/data_model/AuthModel.kt
/*
 * Copyright 2022. Happy coding ! :)
 * Author: Serhii Butryk
 */
package com.serhii.apps.notes.ui.data_model

import com.serhii.apps.notes.control.auth.types.UIRequestType
import javax.crypto.Cipher

data class AuthModel(
    var email: String = "", var password: String = "",
    var confirmPassword: String = "", var authType: UIRequestType = UIRequestType.UN_SET,
) {
    var cipher: Cipher? = null
}